netket_fidelity/operator/singlequbit_gates.py

```diff
@@ -96,34 +82,30 @@ def to_local_operator(self):
         return ctheta - 1j * stheta * spin.sigmax(self.hilbert, self.idx)
 
 
-@partial(jax.vmap, in_axes=(0, None, None), out_axes=(0, 0))
-def get_conns_and_mels_Rx(sigma, idx, angle):
+@partial(jax.vmap, in_axes=(0, None, None, [None, None]), out_axes=(0, 0))
+def get_conns_and_mels_Rx(sigma, idx, angle, local_states):
     assert sigma.ndim == 1
 
-    conns = jnp.tile(sigma, (2, 1))
-    conns = conns.at[1, idx].set(-conns.at[1, idx].get())
+    state_0 = jnp.asarray(local_states[0], dtype=sigma.dtype)
+    state_1 = jnp.asarray(local_states[1], dtype=sigma.dtype)
+
+    cons = jnp.tile(sigma, (2, 1))
```

Comment:
minor, but could you keep this named `conns` everywhere? it's the convention we use throughout netket
```suggestion
    conns = jnp.tile(sigma, (2, 1))
```

Reply:
please fix other occurrences as well.